handler.py

```diff
@@ -25,7 +25,17 @@ def sparkline(datapoints):
 
     return line
 
+def delta(costs):
+    return ' (%+6.2f' % (((costs[-1] - costs[-2])/costs[-2]) * 100.0 ) + '%)'
+
 def report_cost(event, context):
+
+    # Get account alias
+    iam = boto3.client('iam')
+    paginator = iam.get_paginator('list_account_aliases')
+    for aliases in paginator.paginate(PaginationConfig={'MaxItems': 1}):
+        account_name = " for account " + aliases['AccountAliases'][0]
```

Comment:
Instead of tacking the `" for account "` at the beginning of `account_name`, can it be part of the text that's output below? That way `account_name` is just the account name and can be reused for other things maybe in the future.

Reply:
Makes all sense ! As soon as I figure out how to deal with changes to this pull request I'll submit the change :-)

Reply:
Yep, thanks for the reminder!